arch/posix/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ if (CONFIG_GPROF)
   target_link_options(native_simulator INTERFACE "-pg")
 endif()
 
+# These should not be defined by the POSIX architecture. Please remove when drivers are fixed
```

Comment:
This comment is not correct. Please remove it.

Reply:
Please explain how you feel it is not correct

Reply:
Nevermind. I'll remove it and file a bug